src/Nox.Types.EntityFramework/Types/Email/EmailDatabaseConfigurator.cs

```diff
@@ -0,0 +1,23 @@
+﻿
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
+using Nox.Solution;
+using Nox.Types.EntityFramework.Abstractions;
+
+namespace Nox.Types.EntityFramework.Types;
+public class EmailDatabaseConfigurator : INoxTypeDatabaseConfigurator
+{
+    public NoxType ForNoxType => NoxType.Email;
+
+    public bool IsDefault => true;
+
+    public void ConfigureEntityProperty(NoxSolutionCodeGeneratorState noxSolutionCodeGeneratorState, EntityTypeBuilder builder, NoxSimpleTypeDefinition property, Entity entity, bool isKey)
+    {
+        builder
```

Comment:
What would be the column type for SqlServer and Postgres? Do we maybe need to set some additional properties like IsUnicode?

Reply:
Do we need to set the length? What is the default length in this case?
Maybe test this with the SampleWebApp and generate Migrations

Reply:
@gjorgji-grgovski-authoritypartners the default value in sql for this column is nvarchar(max). I left it as unicode, but restricted column to 255 characters